components/sandbox/index.js

```diff
@@ -96,6 +96,12 @@ export default class Sandbox extends Component {
 
 				function sendResize() {
 					var clientBoundingRect = document.body.getBoundingClientRect();
+					if ( 0 === clientBoundingRect.height ) {
+						clientBoundingRect.height = document.getElementById( 'content' ).clientHeight;
+					}
+					if ( 0 === clientBoundingRect.width ) {
+						clientBoundingRect.width = document.getElementById( 'content' ).clientWidth;
+					}
```

Comment:
Can you explain a bit why we need these changes? Also, does it have an impact on embed blocks?